src/examples/ui/sticky-header/css/index.css

```diff
@@ -0,0 +1,223 @@
+@import "@10up/component-accordion";
+
+:root {
+	--header-height: 80px; /** Match .site-header-height. Will be updated via JS */
+	--sticky-header-hide-offset: 6px;
+	--scroll-margin-top-offset: calc(1rem + var(--sticky-header-hide-offset));
+
+	@media (max-width: 343px) {
+		--header-height: 120px; /** Match .site-header height. Will be updated via JS */
+	}
+}
+
+html,
+body {
+    height: 100%;
+}
+
+html {
+	scroll-behavior: smooth;
+
+	&::before {
+		background-image: linear-gradient(to bottom, #eee 0% 50%, #333 50%);  /** Match .site-header background-color and .site-footer background-color */
+		background-repeat: no-repeat;
+		background-size: 100%;
+		content: "";
+		height: 100%;
+		position: fixed;
+		top: 0;
+		width: 100%;
+	}
+}
+
+body {
+	min-width: 320px;
+}
+
+:first-child {
+	margin-top: 0;
+}
+
+:last-child {
+	margin-bottom: 0;
+}
+
+:target {
+	scroll-margin-top: 1rem;
+}
+
+:focus {
+	outline: 5px auto Highlight;
+	outline: 5px auto -webkit-focus-ring-color;
+	outline-offset: 0;
+}
```

Comment:
Should already be in the base styles.

Reply:
The base styles override the browser default with `2px dotted currentColor` which I believe is insufficient for contrast. I considered modifying the base styles but thought it might be better to keep it scoped to my demo at least initially. It's my opinion we should rely on the browser default and only add styles where those are inadequate

Reply:
If we are able to use the browser default focus styles I think we should remove the `a:focus` styles as the focus ring is sufficient, imo

Reply:
Can you make the changes so that base is updated? I think its safe to update the focus styles there (remove bad focus styles).

Reply:
I have removed these focus styles and created a separate PR that removes nearly all of the :focus styles from this project. See PR #29 

I have removed nearly all :focus styles from elements where they had mirrored the :hover styles. Imo, these are two very different user cases, the :focus styles are sufficient for marking user interactivity and provide sufficient contrast. The addition of the :hover state to :focus muddies the default :focus highlight and is an unnecessary and possibly unexpected visual change.

I have retained the :focus style that mirrors the :hover style that is a `transform` `scale`